File: vieroShoesAdminApp/app/src/main/java/com/cindodcindy/vieroshoesadmin/view/BottomNavAdmin.java
package com.cindodcindy.vieroshoesadmin.view;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import android.os.Bundle;
import android.view.MenuItem;

import com.cindodcindy.vieroshoesadmin.R;
import com.cindodcindy.vieroshoesadmin.view.fragment.FrChat;
import com.cindodcindy.vieroshoesadmin.view.fragment.FrGetOrder;
import com.cindodcindy.vieroshoesadmin.view.fragment.FrGetPayment;
import com.cindodcindy.vieroshoesadmin.view.fragment.FrListItem;
import com.cindodcindy.vieroshoesadmin.view.fragment.FrUploadItem;
import com.google.android.material.bottomnavigation.BottomNavigationView;

public class BottomNavAdmin extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_bottom_nav_admin);

        //Menampilkan halaman Fragment yang pertama kali muncul
        getFragmentPage(new FrUploadItem());

        /*Inisialisasi BottomNavigationView beserta listenernya untuk
         *menangkap setiap kejadian saat salah satu menu item diklik
         */
        BottomNavigationView bottomNavigation = findViewById(R.id.bottomNavigationView);
        bottomNavigation.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
            @Override
            public boolean onNavigationItemSelected(@NonNull MenuItem item) {

                Fragment fragment = null;

                //Menantukan halaman Fragment yang akan tampil
                switch (item.getItemId()){
                    case R.id.sell_item:
                        fragment = new FrUploadItem();
                        break;

                    case R.id.home_shoes:
                        fragment = new FrListItem();
                        break;

                    case R.id.get_order:
                        fragment = new FrGetOrder();
                        break;
                    case R.id.get_payment:
                        fragment = new FrGetPayment();
                        break;

                    case R.id.chat:
                        fragment = new FrChat();
                        break;
                }
                return getFragmentPage(fragment);
            }
        });
    }

    //Menampilkan halaman Fragment
    private boolean getFragmentPage(Fragment fragment){
        if (fragment != null){
            getSupportFragmentManager()
                    .beginTransaction()
                    .replace(R.id.page_container, fragment)
                    .commit();
            return true;
        }
        return false;
    }
}

File: vieroShoesAdminApp/app/src/main/java/com/cindodcindy/vieroshoesadminnew/view/BottomNavAdmin.kt
package com.cindodcindy.vieroshoesadminnew.view

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.cindodcindy.vieroshoesadminnew.R
import com.cindodcindy.vieroshoesadminnew.view.fragment.*
import com.cindodcindy.vieroshoesadminnew.view.utils.dexterPermissionMultiLocation
import com.google.android.material.bottomnavigation.BottomNavigationView

class BottomNavAdmin : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_bottom_nav_admin)
        dexterPermissionMultiLocation(this)

        //Menampilkan halaman Fragment yang pertama kali muncul
        getFragmentPage(FrUploadItem())

        /*Inisialisasi BottomNavigationView beserta listenernya untuk
         *menangkap setiap kejadian saat salah satu menu item diklik
         */
        val bottomNavigation = findViewById<BottomNavigationView>(R.id.bottomNavigationView)
        bottomNavigation.setOnNavigationItemSelectedListener { item ->
            var fragment: Fragment? = null
            when (item.itemId) {
                R.id.sell_item -> fragment = FrUploadItem()
                R.id.home_shoes -> fragment = FrListItem()
                R.id.get_order -> fragment = FrGetOrder()
                R.id.get_payment -> fragment = FrGetPayment()
                R.id.chat -> fragment = FrChat()
            }
            getFragmentPage(fragment)
        }
    }

    //Menampilkan halaman Fragment
    private fun getFragmentPage(fragment: Fragment?): Boolean {
        if (fragment != null) {
            supportFragmentManager
                    .beginTransaction()
                    .replace(R.id.page_container, fragment)
                    .commit()
            return true
        }
        return false
    }
}